CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### 2.4.2 (Unreleased)
 
+- [#563](https://github.com/influxdata/clockface/pull/563): Fixes issue with Rollup copy plugin where src and destination could be the same, causing development issues.
```

Comment:
I don't think we need this in the changelog since it's a development issues. We usually only put user-facing changes in changelogs